x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/eql/build_alert_group_from_sequence.ts

```diff
@@ -139,19 +187,34 @@ export const buildAlertGroupFromSequence = (
     }
   );
 
-  return [...wrappedBuildingBlocks, sequenceAlert];
+  // sequence alert guaranteed to be first
+  return [sequenceAlert, ...wrappedBuildingBlocks] as [
```

Comment:
This cast shouldn't be necessary

Reply:
Fixed here: https://github.com/elastic/kibana/pull/189725/commits/3079b9eabcd2274281f1678704f1b09c5cc0e655